src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementKeyValueAdapter.cs

```diff
@@ -195,18 +197,60 @@ public FeatureManagementKeyValueAdapter(FeatureFilterTracing featureFilterTracin
 
                 string telemetryPath = $"{featureFlagPath}:{FeatureManagementConstants.Telemetry}";
 
-                if (telemetry.Enabled)
-                {
-                    keyValues.Add(new KeyValuePair<string, string>($"{telemetryPath}:{FeatureManagementConstants.Enabled}", telemetry.Enabled.ToString()));
-                }
-
                 if (telemetry.Metadata != null)
```

Comment:
Should we move this inside the `if (telemetry.Enabled)`? No one cares about it if telemetry is not enabled.

Reply:
I see now, I thought there was a scenario in feature management where it could still be used but I don't see how. I can move it inside the enabled block too.